app/src/components/PersonCredentialSpinner.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTheme } from '@hyperledger/aries-bifold-core'
```

Comment:
Curious if there was a component in Bifold that could just be modified to take in a different graphic?